2-farm/lessons/1-predict-plant-growth/README.md

```diff
@@ -265,6 +265,7 @@ Read more on digital agriculture on the [Digital Agriculture Wikipedia page](htt
 
 The full growing degree days calculation is more complicated than the simplified one given here. Read more about the more complicated equation and how to deal with temperatures below the baseline on the [Growing Degree Day Wikipedia page](https://wikipedia.org/wiki/Growing_degree-day).
 
+[Hi-Tech Farms of Future](https://www.youtube.com/watch?v=KIEOuKD9KX8)
```

Comment:
Can you expand on what this video is. For example, here's a video link in [one of the lessons](https://github.com/microsoft/IoT-For-Beginners/blob/main/4-manufacturing/lessons/3-run-fruit-detector-edge/README.md).

```md
* Learn more about running AI services in IoT Edge by watching the [learn how to use Azure IoT Edge on a pre-built AI service on the Edge to do language detection episode of Learn Live on Microsoft Channel9](https://channel9.msdn.com/Shows/Learn-Live/Sharpen-Your-AI-Edge-Skills-Episode-4-Learn-How-to-Use-Azure-IoT-Edge-on-a-Pre-Built-AI-Service-on-t?WT.mc_id=academic-17441-jabenn)
```